---
permalink: /softwares/
layout: "single"
author_profile: true
title: Softwares
---

<!--aPCoA-->
<div style="width: 100%; margin-top:1.5em; text-align: left;">
    <details>
    <summary><h2 style="display: inline;">aPCoA</h2></summary>
    <dl3 style="margin-top:1.5em; text-align: left;">
        <p>Covariate Adjusted Principal Coordinates Analysis: <a href="https://biostatistics.mdanderson.org/shinyapps/aPCoA/" target="_blank">aPCoA</a> 
        is an easy-to-use tool, available as both an R package and a Shiny app, <br/>
        to improve data visualization in this context, enabling enhanced presentation of the effects of interest.</p>
    </dl3>
    </details>
</div>

<!--BAZE-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">BAZE</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Bayesian zero-constrained regression with compositional covariates: <a href="https://github.com/LyonsZhang/BAZE" target="_blank">BAZE</a><br/> 
     The proposed Bayesian sparse regression model addressed the challenges of microbiome data, <br/>
     including the compositional nature of the data, the high dimension, and the relatedness among the features. <br/>
     This packages include two different ways of addressing the fixed-sum constraint: the constrast transformation and the generalized transformation.</p>
</dl3>
</details>
</div>

<!--CAMLU-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">CAMLU</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Cell Annotation using Machine Learning-based method for the presence of Unknown cells: <a href="https://github.com/ziyili20/CAMLU.git" target="_blank">CAMLU</a> 
    is an R package <br/> 
    that provides an autoencoder based method for annotating cell types from scRNA-seq data. <br/>
    The function can identify unknown cells with the input training data. <br/>
    It also can annotate the full lists of cell types with consideration of unknown cell types. <br/>
    This vignette introduces the CAMLU function and the things it can do for you. </p>
</dl3>
</details>
</div>

<!--DINGO, iDINGO-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">DINGO, iDINGO</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Integrative Differential Network Analysis in Genomics: <a href="https://github.com/cran/iDINGO" target="_blank">iDINGO</a> 
     is a pathway-based method <br/>
     for estimating group-specific conditional dependencies and inferring differential networks between groups, based on genomic data. <br/>
     This can be done in a single-platform framework (for example, RNA-Seq data) or an integrative multi-platform framework<br/>
     (microRNA -> RNA -> Proteomics, where data from all three platforms are available for every sample).<br/>
     We recommend filtering genomic data to fewer than 300 genes, generally filtered using a pathway/pathways of interest. <br/>
     Single-platform analyses are run using DINGO with an nxp matrix, where n is the number of samples.</p>
</dl3>
</details>
</div>

<!--GENECLUST-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">GENECLUST</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p><a href="https://odin.mdacc.tmc.edu/~kim/geneclust/" target="_blank">GENECLUST</a> 
      is a piece of computer software which can be used as a tool for exploratory analysis of gene expression microarray data. <br/>
      The development of GeneClust was motivated by surging interest to search for interpretable biological structure in gene expression microarray data.</p>
</dl3>
</details>
</div>

<!--looPA-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">looPA</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Microbiome feature identification using leave-one-out with Permanova assessment: <a href="https://rdrr.io/github/YushuShi/looPA/" target="_blank">looPA</a>
      is a permutation based method, <br/>
      which can account for phylogenetic relatedness between taxonomic features
      and identify important features for further investigation.</p>
</dl3>
</details>
</div>

<!--mediateR-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">mediateR</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>A unified mediation analysis framework for integrative cancer proteogenomics with clinical outcomes: <a href="https://github.com/longjp/mediateR" target="_blank">mediateR</a>
    </p>
</dl3>
</details>
</div>

<!--NExUS-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">NExUS</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Network Estimation across Unequal Sample sizes: <a href="https://github.com/priyamdas2/NExUS" target="_blank">NExUS</a> 
     is a Bayesian method that enables joint learning of multiple networks <br/>
     while avoiding artefactual relationship between sample size and network sparsity.</p>
</dl3>
</details>
</div>

<!--PRECISE-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">PRECISE</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Personalized Cancer-specific Integrated Network Estimation: <a href="https://github.com/MinJinHa/PRECISE" target="_blank">PRECISE</a> 
     is a general Bayesian framework for integrating existing interaction databases, <br/>
     data-driven de novo causal structures,
     and upstream molecular profiling data to estimate cancer-specific integrated networks, <br/>
     infer patient-specific networks and elicit interpretable pathway-level signatures.</p>
</dl3>
</details>
</div>

<!--ProgPerm-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">ProgPerm</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Progressive permutation for a dynamic representation of the robustness of microbiome discoveries: <a href="https://biostatistics.mdanderson.org/shinyapps/ProgPerm/" target="_blank">ProgPerm</a><br/> 
     We have developed this into a user-friendly and efficient R-shiny tool with visualizations.<br/> 
     By default, we use the Wilcoxon rank sum test to compute the p-values, since it is a robust nonparametric test.<br/> 
     Our proposed method can also utilize p-values obtained from other testing methods, such as DESeq. <br/>
     This demonstrates the potential of the progressive permutation method to be extended to new settings.</p>
</dl3>
</details>
</div>

<!--sparseMbClust-->
<div style="margin-top:1.5em; text-align: left;">
<details>
<summary><h2 style="display: inline;">sparseMbClust</h2></summary>
<dl3 style="margin-top:1.5em; text-align: left;">
    <p>Sparse tree-based clustering of microbiome data: <a href="https://github.com/YushuShi/sparseMbClust/" target="_blank">sparseMbClust</a><br/> 
     R and Matlab code to implement the methods in "Sparse tree-based clustering of microbiome data <br/>
     to characterize microbiome heterogeneity in pancreatic cancer</p>
</dl3>
</details>
</div>
